src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -94,6 +97,11 @@ export type OpportunitiesDataFull = {
 
 export const selectPortfolioAccounts = (state: ReduxState) => state.portfolio.accounts.byId
 
+export const validatorFromAccountSpecifier = (accountSpecifier: AccountSpecifier) => {
+  if (accountSpecifier.includes('osmosis')) return SHAPESHIFT_OSMO_VALIDATOR_ADDRESS
+  else return SHAPESHIFT_VALIDATOR_ADDRESS
```

Comment:
can you please do an `F2` (rename symbol) and make this `SHAPESHIFT_COSMOS_VALIDATOR_ADDRESS` now, as previously we only had one, but now we have OSMO it's not clear what this other one is